charts/nginx-gateway-fabric/values.yaml

```diff
@@ -83,33 +118,64 @@ nginx:
     # -- The NGINX image to use.
     repository: ghcr.io/nginxinc/nginx-gateway-fabric/nginx
     tag: edge
+    # @schema
+    # enum:
+    #   - Always
+    #   - IfNotPresent
+    #   - Never
+    # @schema
     pullPolicy: Always
 
   # -- Is NGINX Plus image being used
   plus: false
 
+  # @schema
+  # type: object
+  # properties:
+  #   disableHTTP2:
+  #     type: boolean
+  #   ipFamily:
+  #     type: string
+  #     enum:
+  #       - ipv4
+  #       - ipv6
+  #       - dual
+  #   telemetry:
+  #     type: object
+  #     properties:
+  #       exporter:
+  #         type: object
+  #         properties:
+  #           endpoint:
+  #             type: string
+  #             format: idn-hostname
+  #           interval:
+  #             type: string
+  #           batchSize:
+  #             type: integer
+  #           batchCount:
+  #             type: integer
+  #       serviceName:
+  #         type: string
+  #       spanAttributes:
+  #         type: array
+  #         items:
+  #           type: string
```

Comment:
We have restrictions for these defined in the CRD schema for NginxProxy, we should use those for consistency.

Reply:
Updated with restrictions in the CRD